workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+### Step 2: Create Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within the query.
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. The way in which you aggregate a metric is contingent on the initial metric you have selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. The below example is grouping the `container.name` metric by all unique instances of the `container.name` metric:
+
+![](https://lh7-us.googleusercontent.com/lJ_3zm4C37Zgv8oWBRza4Z9-BbdJdlXD-lxbFq2qfGD579JQ9tNR_1MbC0zLLLW9ufQ-MQQa1_nwD4tg5PG-9QTpCDdsX5rE4UDIN-y9sILxyPlCHF8C8FcYv1od9VqK2ZACHS1NBAQFDk6yuOLgRzM)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
+| :------------------- | :------------------------------------------------------------ |
+| __Average By__       | Average of all values in selected time range for given metric |
+| __Sum By__           | Sum of all values in selected time range for given metric |
+| __Minimum By__       | Minimum of all values in selected time range for given metric |
+| __Maximum By__       | Maximum of all values in selected time range for given metric |
+| __Any By__           | Gathers all possible values in selected time range for given metric __and/or__ attribute |
+| __Unique By__        | Count of unique values in selected time range for given metric __and/or__ attribute |
+| __Count By__         | Count of all values in selected time range for given metric __and/or__ attribute |
+| __Group By__         | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Inner Value Aggregation
+
+The Inner Value Aggregation is a nested aggregation technique that allows you to aggregate your initial query. The below example is grouping the `k8s.node.memory.usage` metric by summing the `k8s.cluster.name` attributes together and aggregating __Any__ `k8s.node.memory.usage`:
+
+![](https://lh7-us.googleusercontent.com/fo5Img7dsK14nN-w6atI390lqE-Lrg9oHS6BlqlgLwJKjNjpke85oDeEVYfhbl3CGj3hPiZ6mfnK5XXGPZGlxbfzY5wjd31myEhAR4gAJnQgB5Eii1Ex7CvkQgufEmUNFVcKZiLkUBFt35WBIGRtBHc)
+
+#### Filter
+
+The filter feature is used to remove unwanted metrics within your query. The below example is filtering out any `host.id` containing `ip-172-31-13-218.us-east-2.compute.internal`:
```

Comment:
Worth calling out that filters are per metric, and then having a list of supported filter functions.